esmvalcore/preprocessor/_area.py

```diff
@@ -349,6 +349,26 @@ def _select_representative_point(shape, lon, lat):
     return select
 
 
+def _correct_coords_from_shapefile(cube, cmor_coords):
+    """Get correct lat and lon from shapefile."""
+    lon = cube.coord(axis='X').points
+    lat = cube.coord(axis='Y').points
+    if cube.coord(axis='X').ndim < 2:
+        lon, lat = np.meshgrid(lon, lat, copy=False)
+
+    if not cmor_coords:
+        # Wrap around longitude coordinate to match data
+        lon = lon.copy()  # ValueError: assignment destination is read-only
+        lon[lon >= 180.] -= 360.
+        # the NE mask has no points at x = -180 and y = +/-90
```

Comment:
The function `extract_shape` should work with any shapefile, so this very specific natural earth shapefile stuff should not be here, see also my earlier comment: https://github.com/ESMValGroup/ESMValCore/pull/319#discussion_r360447490

Reply:
correct! I added a check see if we have NE file or not :grin:

Reply:
it **has** to be there, unless we edit the NE file in advance and add data via some extrapolation procedure, but we dont do that; otherwise we end up with a white 1-deg wide tear in the middle of the blue Atlantic

Reply:
Can you share the recipe or diagnostic or add a test that produces this 1-deg wide tear? I would like to reproduce your problem.

Reply:
run with any of our current shapefiles in `ne_masks` and take the bit of code that eliminates the tear. Come on guys, this is old stuff that's been reviwed and tested aeons ago